tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_prices.feature

```diff
@@ -2,7 +2,6 @@
 @restore-products-before-feature
 @clear-cache-before-feature
 @product-combination
-@update-product-prices
```

Comment:
This tag was actually added to easily launch all the features related to prices, whether it's from a combination or regular product
It's convenient during the development phase and debugging to check everything is fine since all those features may be related without realizing it So I think it should stay here and you should add it to `update-prices-without-states.feature` as well